config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/config.yaml

```diff
@@ -88,10 +88,24 @@ presubmits:
         # docker-in-docker needs privileged mode
         securityContext:
           privileged: true
+        volumeMounts:
+          - name: ssh
+            mountPath: /root/.ssh
+            readOnly: true
+            items:
+              - key: ssh-private
+                path: google_compute_engine
+              - key: ssh-public
+                path: google_compute_engine.pub
       resources:
         requests:
           # these are both a bit below peak usage during build
           # this is mostly for building kubernetes
           memory: "9000Mi"
           # during the tests more like 3-20m is used
           cpu: 2000m
+      volumes:
+        - name: ssh
+          secret:
+            defaultMode: 0400
+            secretName: ssh-key-secret
```

Comment:
Why do no other jobs need this?

Reply:
that knowledge has been lost with @hantaowang, are you still around to answer?

Reply:
jobs that need to provision gce/gke clusters will need this secret.

Reply:
this is for dumping kubelet/master logs from the cluster

Reply:
My queston is I do not see any jobs under https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cloud-provider/gcp/sig-gcp-gce-config.yaml directly referencing this secret.  How are they getting the secret configured and can we do the same thing?

Reply:
GKE needed this specifically because kubetest needs this inorder to use the GKE deployment. Because we are handling cluster up and down outside of kubetest, is this needed here?

Also, test-infra here has an additional [script](https://github.com/kubernetes/test-infra/blob/master/scenarios/kubernetes_e2e.py#L226) that sets this up, so we shouldnt need to to it to external test-infra.

Reply:
I am not actually sure how that script is invoked, make @krzyzacy knows. You will probably at the very least need to add the `preset-k8s-ssh: "true"` label though.

Reply:
@krzyzacy we also have a new test that ssh's into the node to check some things during the course of the test which is failing because the keys aren't there. Wondering if I can just put `preset-k8s-ssh` to fix that or if I need to do the manual mount

Reply:
I think your script doesn't have https://github.com/kubernetes/test-infra/blob/master/scenarios/kubernetes_e2e.py#L226-L239 equivalent

you can add that somewhere and use the preset, if you want